packages/wdio-utils/src/shim.js

```diff
@@ -146,14 +146,21 @@ const executeAsync = async function (fn, retries, args = []) {
  * shim to make sure that we only wrap commands if wdio-sync is installed as dependency
  */
 try {
-    // eslint-disable-next-line import/no-unresolved
-    const wdioSync = require('@wdio/sync')
-    hasWdioSyncSupport = true
-    runFnInFiberContext = wdioSync.runFnInFiberContext
-    wrapCommand = wdioSync.wrapCommand
-    executeHooksWithArgs = wdioSync.executeHooksWithArgs
-    executeSync = wdioSync.executeSync
-    runSync = wdioSync.runSync
+    /**
+     * only require `@wdio/sync` if `WDIO_NO_SYNC_SUPPORT` which allows us to
+     * create a smoke test scenario to test actual absence of the package
+     * (internal use only)
+     */
+    if (!process.env.WDIO_NO_SYNC_SUPPORT) {
+        // eslint-disable-next-line import/no-unresolved
+        const wdioSync = require('@wdio/sync')
```

Comment:
should be here `const wdioSync = require('@wdio/sync').default` ?

Reply:
This would give me the reference to the [`sync`](https://github.com/webdriverio/webdriverio/blob/cb-retry-fix/packages/wdio-sync/src/index.js#L79-L83) method. I need to exported functions above though.